plugins/node/opentelemetry-instrumentation-generic-pool/src/instrumentation.ts

```diff
@@ -40,7 +40,7 @@ export default class Instrumentation extends InstrumentationBase {
     return [
       new InstrumentationNodeModuleDefinition(
         MODULE_NAME,
-        ['>=3'],
+        ['^3.0.0'],
```

Comment:
nit: Similar to the amqplib point above. This PR is slightly more than just a docs PR.